scripts/release-lib.sh

```diff
@@ -52,8 +54,8 @@ function yesno() {
 
 #-------------------------------------------------------------------------------
 function check_milestone() {
-  if yesno "Visit the milestones page (https://github.com/Kong/kong/milestone) and ensure PRs are merged. Press 'y' to open it or Ctrl-C to quit"; then
-    $browser https://github.com/Kong/kong/milestones
+  if yesno "Visit the milestones page (https://github.com/$GITHUB_ORG/kong/milestone) and ensure PRs are merged. Press 'y' to open it or Ctrl-C to quit"; then
```

Comment:
Probably a lot of these functions can go back to the release.sh file, now that there's only one file and there's no need to share functionality across 3 files any more

Reply:
I agree.  If the split between top-level code in `make-release` and implementation in `release-lib.sh` becomes too bothersome, we can still merge the two into one.